sys/scripts/createSummary.py

```diff
@@ -178,7 +178,7 @@ def parseFile(log_file):
       current_state = "END"
       current_buffer = ""
       current_test = testResult()
-      for line in open(log_file,'r'):
+      for line in open(log_file,'r', encoding='utf-8', errors='ignore'):
```

Comment:
I think you want to re-diff. This change has been already merged via commit 4c62ffca98d1929995139b335b6d204a385d9242 (Pull Request #609).